tests/phpunit/admin/class-wc-stripe-express-checkout-controller-test.php

```
@@ -22,12 +22,12 @@ public function setUp(): void {
 		parent::setUp();
 
 		// Setup existing keys
-		$settings                         = WC_Stripe_Helper::get_stripe_settings();
+		$settings                         = WC_Stripe::read_settings_option();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe' in method 'setUp'. (undefined)

(StaticAccess)